src/Main.java

```diff
@@ -1,5 +1,91 @@
 public class Main {
     public static void main(String[] args) {
 
+        System.out.println("Задача № 1");
+
+        int clientOS = 1;
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для IOS по ссылке");
+        } else {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+        System.out.println("Задача № 2");
+        // Исправлено: Вывод сообщения соответствует шаблону.
+
+        int clientOS2 = 1;
```

Comment:
можно было не заводить новую переменную, а использовать уже созданную
```suggestion
        clientOS = 1;
```